src/app/components/details/details.component.scss

```diff
@@ -0,0 +1,88 @@
+//Details-Card
```

Comment:
@tarun-chakravarthy (future Me)
Add all the common styles to the global scss file, Import, and use them.